frontend/src/app/register/register.ctrl.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Controller for register page
+ */
+angular.module('flocs.user')
+.controller('registerCtrl', ['$scope','$location','$log','userDao', 
+    function($scope,$location,$log,userDao){
+
+	    function send(){
```

Comment:
We suggested the renaming, because when you look at the corresponding html form, you just see that click on submit sends something somewhere, which is not super descriptive. We thought that register() is more clear, because when you submit the registration form, the controller tries to register you.